modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc

```diff
@@ -82,20 +194,39 @@ Redpanda uses the following default ports:
 | Admin API
 |===
 
-Minimum 10 GigE.
+To change these ports, see xref:manage:kubernetes/networking/configure-listeners.adoc[].
 
-Redpanda recommends using NodePorts instead of Loadbalancers. See the xref:./kubernetes-best-practices.adoc#external-networking[external networking best practices].
+Depending on your deployment and security policies, you may not be able to access worker nodes through a NodePort Service.
+If you choose to use another Service, consider the impact on the cost and performance of your deployment:
 
-== Tuning
+* LoadBalancer Service - To make each Redpanda broker accessible with LoadBalancer Services, you need one LoadBalancer Service for each Redpanda broker so that requests can be routed to specific brokers rather than balancing requests across all brokers. Load balancers are expensive, add latency and occasional packet loss, and add an unnecessary layer of complexity.
+* Ingress - To make each Redpanda broker accessible with Ingress, you need to run an Ingress controller and set up routing to each Redpanda broker. Routing adds latency and can be a throughput bottleneck.
 
-Before deploying Redpanda to production, each worker node that runs Redpanda must be tuned to optimize the Linux kernel for Redpanda processes.
+See xref:manage:kubernetes/networking/networking-and-connectivity.adoc[Networking and Connectivity].
 
-See xref:./kubernetes-tune-workers.adoc[Tuning Kubernetes Worker Nodes for Production].
+=== Use ExternalDNS for external access
 
-== Sizing
+*Recommendation*: Redpanda Data recommends using ExternalDNS to manage DNS records for your Pods' domains. ExternalDNS synchronizes exposed Kubernetes Services with various DNS providers, rendering Kubernetes resources accessible through DNS servers.
 
-For help sizing your Kubernetes cluster, see xref:deploy:deployment-option/self-hosted/manual/sizing.adoc[Sizing Guidelines].
+Benefits of ExternalDNS include:
+
+* *Automation*: ExternalDNS automatically configures public DNS records when you create, update, or delete Kubernetes Services or Ingresses. This eliminates the need for manual DNS configuration which can be error-prone.
```

Comment:
*Automation*: ExternalDNS automatically configures public DNS records when you create, update, or delete Kubernetes Services or Ingresses. This eliminates the need for manual DNS configuration, which can be error-prone.